src/cockpit/packages.py

```diff
@@ -227,6 +227,18 @@ def check(self, path_exists: Callable[[str], bool]) -> bool:
         return self.exists == path_exists(self.path)
 
 
+class DisjunctiveCondition(Condition):
```

Comment:
The absolute trivialness of the commit which adds this class is sort of a vindication of everything that came before.  Thanks for keeping at it.